### PR TITLE
This adds a button to delete all comments at once 

### PR DESCRIPTION
**Pull request by Aradhya-B**
### Comments
image

### After clicking delete all
image

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/comments.js [hunk 1]
```diff
@@ -15,6 +15,16 @@ async function fetchAndInsertDataIntoDOM() {
...
 }
 
+/*
+ * Deletes all comments from Datastore and 
+ * calls function to refresh comment page view
+ */
+function deleteAllCommentsFromStorage() {
+	fetch('delete-data', {
+		method: 'POST'
+	}).then(_ => this.fetchAndInsertDataIntoDOM())
```
Comment (nickKelman) [inline portfolio/src/main/webapp/scripts/comments.js]: Interesting, I haven't seen the _ notation before. Is this using the underscore library to declare a function?

**Review by nickKelman — APPROVED** [review]
Very clean, nice work!

File: portfolio/src/main/webapp/scripts/comments.js (hunk 1 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/scripts/comments.js]: No, it's not using the underscore library, it's just a notation that I picked up to use in promise chains when I don't need to use whatever resolved from the last thing in the chain. Not sure if it's common but I saw it all over the place during my last internship.

Comment (nickKelman) [reply portfolio/src/main/webapp/scripts/comments.js]: Ah ok. I think typically we use this notation in our codebase for defining annonymous functions:

() => this.fetchAndInsertDataIntoDOM()

I might be misunderstanding what the underscore does though

Comment (Aradhya-B) [reply portfolio/src/main/webapp/scripts/comments.js]: Interesting, I'll take to using that notation then from now on. The underscore doesn't actually do anything haha.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    response.setContentType("application/json");
+    response.getWriter().println("");
+  }
+
+  private void deleteCommentById(String idString) {
+    long id = Long.parseLong(idString);
+    Key commentKey = KeyFactory.createKey("Comment", id);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (cftorres) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java]: Might want to extract this out into an instance variable so you can use the same instance throughout here

**Review by cftorres — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java (hunk 2 above)
Comment (Aradhya-B) [reply portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java]: Good point, will make that change.

**Pull request merged**